src/mon/MonCommands.h

```diff
@@ -441,6 +441,27 @@ COMMAND("mon remove " \
 COMMAND("mon rm " \
 	"name=name,type=CephString", \
 	"remove monitor named <name>", "mon", "rw", "cli,rest")
+COMMAND("mon features list " \
+        "name=feature_type,type=CephChoices,strings=all|monmap,req=false " \
+        "name=with_value,type=CephChoices,strings=--with-value,req=false", \
+        "list available mon map features to be set/unset", \
+        "mon", "r", "cli")
+COMMAND("mon features set " \
+        "name=feature_name,type=CephString " \
+        "name=feature_type,type=CephChoices,strings=persistent|optional,req=false " \
```

Comment:
this interface doesn't make sense to me. any given feature is either persistent or not; having the user pass that as an argument is just an opportunity for them to get it wrong (the mon already knows which it is).  (unless it's meant as a safety check?)

I'd suggest instead

 ceph mon features set <optional feature>
 ceph mon features set-persistent <persistent feature> [--force]

Or perhaps 'feature' instead of  'features'.  :/